### ISSUE TITLE
Issue API magento getting image's base64 code

### ISSUE THREAD

**Issue by aurquiel** [magento/magento2#30174]
API magento 2.4 get image of product

Hello i am trying to get the base64 image's code from the api using this endpoints

**Preconditions:**

Magento 2.4-develop
Php 7.4
MYSQL 8.0

**Manual testing scenario:**

1. Add image to a product by API
**POST** **`/V1/products/{sku}/media`** -> Execute

![Screenshot from 2021-01-12 15-32-50](https://user-images.githubusercontent.com/51679138/104321295-19650d00-54ec-11eb-8efa-c6c5b5fdea16.png)

**Actual result:** :heavy_check_mark:  An image was successfully added to the product

![Screenshot from 2021-01-12 15-35-08](https://user-images.githubusercontent.com/51679138/104321724-ac9e4280-54ec-11eb-88cb-91183de00f7e.png)
![Screenshot from 2021-01-12 15-35-35](https://user-images.githubusercontent.com/51679138/104321918-ebcc9380-54ec-11eb-824f-77a6f00ca7d9.png)

2. Send **GET** **`/V1/products/{sku}/media`**

**Actual Result:** :heavy_multiplication_x: Base64 code of image is absent

```json
[
  {
    "id": 1,
    "media_type": "image",
    "label": "Image",
    "position": 1,
    "disabled": false,
    "types": [
      "image",
      "small_image",
      "thumbnail"
    ],
    "file": "/t/e/test.jpeg"
  }
]
```

3. Send **GET** **`/V1/products/{sku}/media/{entryId}`**

**Actual Result:** :heavy_multiplication_x: Base64 code of image is absent

```json
{
  "id": 1,
  "media_type": "image",
  "label": "Image",
  "position": 1,
  "disabled": false,
  "types": [
    "image",
    "small_image",
    "thumbnail"
  ],
  "file": "/t/e/test.jpeg"
}
```
I added de 1 as id in the second endpoint i used but i get the same answer

**Expected Result:** :heavy_check_mark: Base64 code of image is present

Documentation

![2021-01-12_15-49](https://user-images.githubusercontent.com/51679138/104322712-f0457c00-54ed-11eb-806a-268d859d13b1.png)

There is a field i am not getting in the json

"content": { "base64_encoded_data": "string", "type": "string", "name": "string" },

- [ ] Severity: **S0** _- Affects critical data or functionality and leaves users with no workaround._

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-Alfa 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-40378` were created

**Issue Available**: @engcom-Alfa, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by magento-engcom-team** [comment]
Unfortunately, we are archiving this ticket now as it did not get much attention from both Magento Community and Core developers for an extended period. This is done in an effort to create a quality, community-driven backlog which will allow us to allocate the required attention more easily.

Please feel free to comment or reopen according to the [Issue reporting guidelines](https://github.com/magento/magento2/wiki/Issue-reporting-guidelines) 
the ticket if you are still facing this issue on the latest `2.x-develop` branch. Thank you for collaboration.

**Comment by magento-engcom-team** [comment]
Hi @aurquiel. 

Thank you for your report and collaboration!

The related internal Jira ticket `MC-40378` was closed as `Fixed`. 

The fix will be available with the upcoming `2.4.3` release.
